pyart/aux_io/__init__.py

```diff
@@ -42,5 +42,6 @@
 from .odim_h5 import read_odim_h5
 from .gamic_hdf5 import read_gamic
 from .sinarame_h5 import read_sinarame_h5
+from .rainbow_wrl import read_rainbow_wrl
```

Comment:
The `read_rainbow_wrl` function should be added to the docstring at the top of this module so that the function is included in the User Reference Manual.